defaults/main.yml

```diff
@@ -21,9 +21,20 @@ monasca_wait_for_period: 10
 monasca_log_dir: /var/log/monasca
 monasca_log_level: WARN
 monasca_jar_dir: /opt/monasca
-mysql_db: mon
+database_name: mon
 database_type: influxdb
+hibernate_support_enabled: "false"
+database_port: 3306
+database_host: 192.168.10.4
```

Comment:
The readme needs to be updated to reflect the new names for some of these variables.